src/components/table/helpers/mixin-sort.js

```diff
@@ -0,0 +1,60 @@
+import { EVENT_THEAD_CLICKED } from "./constants"
```

Comment:
Delete this file. ;-) It's the old namespaced one in the helpers folder. The correct one is the one in the mixins folder, was we discussed earlier today.

Reply:
gone, thanks